src/assembler/elf/writer.ts

```diff
@@ -153,7 +153,7 @@ export class FileWriter {
   }
 
   /**
-   * Aligns the content to be specified alignment.
+   * Aligns the content to be specified alignment by filling with tbd
```

Comment:
Why is here a tbd and TODO?
Is this "allowed" in the master?

Reply:
made a new issue and removed it: https://virtualctboard.atlassian.net/browse/VCB-208